.github/workflows/average-metrics.yaml

```diff
@@ -0,0 +1,32 @@
+name: Average Metrics
+
+on:
+  workflow_dispatch:
+  pull_request:
+  schedule:
+    - cron: "0 4 * * 1"
+
+jobs:
+  averageWeek:
+    name: Average Metrics
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Post Request
+        env:
+          INTERNAL_AVGERAGE_SECRET: ${{ secrets.INTERNAL_AVERAGE_SECRET }}
```

Comment:
There's a typo here. Change `INTERNAL_AVGERAGE_SECRET` to `INTERNAL_AVERAGE_SECRET` as on line 21.